### DISCUSSION TITLE
Oh, what happened?

### DISCUSSION THREAD

**Discussion by trip5** [lubeda/EsphoMaTrix#96] [General]
I was watching your repo to see your progress on the next beta... the one with two fonts.

And the branch has disappeared.  Could you use some help?  I'm not great at coding but not terrible either.  It's like pounding my head on a brick wall at times until I get what I want.

**Comment by lubeda** [comment]
Hi,
since the "[old](https://github.com/lubeda/EsphoMaTrix)" api to control the display is very different to the new (work in progress) one, i decided to leave the old repo as it is and started a new [repo](https://github.com/lubeda/EspHoMaTriXv2).

The new version is working for me but not ready to promote, You will find the features (two fonts, better naming of the functions, flexible display of nothing, the clock etc.) of the disappeared branch in the new repo. 

So development is still alive but only in the new branch.

**Comment by trip5** [comment]
Whew.  I took a look at your readme, not the code just yet.  You've been busy!  I can see why this is a V2!

I think I have to buy another Ulanzi just to play with since the wife has already gotten used to the Ulanzi in the bedroom and making it unstable might result in trouble.  Do you have an Ulanzi TC-whatever?  I suspect this device may prove to be the most popular... it can be had for $50-60 on Aliexpress... makes me sad for my old Awtrix Mini since it was more than double the price, looks less professional, and I can't even replace the messed-up pixel display (faded LEDs) since no one makes those mini 8x32's anymore.  And if they do, it still seems better to buy an Ulanzi...

Anyways good to know and I'll see you over at V2!
